lib/streamlit/elements/markdown.py

```diff
@@ -17,6 +17,7 @@
 from streamlit import type_util
 from streamlit.proto.Markdown_pb2 import Markdown as MarkdownProto
 from .utils import clean_text
```

Comment:
heads up that this will need a small change as I removed `elements.utils.clean_text` in favor of `string_util.clean_text` in #5121

Reply:
Addressed.